hack/dockerfile/install-binaries.sh

```diff
@@ -47,10 +47,31 @@ install_proxy() {
 }
 
 install_dockercli() {
-	echo "Install docker/cli version $DOCKERCLI_COMMIT"
-	git clone "$DOCKERCLI_REPO" "$GOPATH/src/github.com/docker/cli"
-	cd "$GOPATH/src/github.com/docker/cli"
-	git checkout -q "$DOCKERCLI_COMMIT"
+	DOCKERCLI_CHANNEL=${DOCKERCLI_CHANNEL:-edge}
+	DOCKERCLI_VERSION=${DOCKERCLI_VERSION:-17.06.0-ce}
+	echo "Install docker/cli version $DOCKERCLI_VERSION from $DOCKERCLI_CHANNEL"
+
+	arch=$(uname -m)
+	# No official release of these platforms
+	if [[ "$arch" != "x86_64" ]] && [[ "$arch" != "s390x" ]] && [[ "$arch" != "armhf" ]]; then
```

Comment:
The output of `uname -m` will likely never be `armhf` -- it'll be something like `armv5tel`, `armv6l`, `armv7l`, `armv8l`, `aarch64`, etc.  Using `go env GOARCH` or `go env GOHOSTARCH` will likely be more reliable.